Document GHC-00711 (#434)
haskellfoundation/error-message-index: message-index/messages/GHC-00711/index.md

---
title: Duplicate warning declarations
summary: Duplicate deprecation warnings are illegal
severity: error
introduced: 9.6.1
---

> Using a deprecation pragma it is possible to declare that a Haskell entity such as a type or a toplevel function is deprecated.

Reviewer: ```suggestion
A Haskell entity, such as a type or a top-level definition, can be declared to be deprecated using a pragma.
```
or
```suggestion
Using a deprecation pragma, it is possible to declare that a Haskell entity such as a type or a top-level function is deprecated.
```

I prefer the former fix, as I feel that it reads a bit less repetitively. But either fix is just fine - pick what you prefer!
